app/models/importers/hmis_auto_migrate/base.rb

```diff
@@ -97,11 +99,7 @@ def upload(file_path:)
     end
 
     private def add_content_to_upload_and_save(file_path:)
-      path = Pathname.new(file_path)
-      @upload.hmis_zip.attach(
```

Comment:
maybe this change isn't necessary but seems like newer active storage supports a less awkward syntax

Reply:
As long as we can confirm we're setting `filename` I think that's great, way cleaner.